libmpv/csharp/Form1.cs

```diff
@@ -0,0 +1,167 @@
+        private void LoadMpvDynamic()
+        {
+            _libMpvDll = LoadLibrary("mpv-1.dll"); // The dll is included in the DEV builds by lachs0r: https://mpv.srsfckn.biz/
+            _mpvCreate = (MpvCreate)GetDllType(typeof(MpvCreate), "mpv_create");
+            _mpvInitialize = (MpvInitialize)GetDllType(typeof(MpvInitialize), "mpv_initialize");
+            _mpvTerminateDestroy = (MpvTerminateDestroy)GetDllType(typeof(MpvTerminateDestroy), "mpv_terminate_destroy");
+            _mpvCommand = (MpvCommand)GetDllType(typeof(MpvCommand), "mpv_command");
+            _mpvSetOption = (MpvSetOption)GetDllType(typeof(MpvSetOption), "mpv_set_option");
+            _mpvSetOptionString = (MpvSetOptionString)GetDllType(typeof(MpvSetOptionString), "mpv_set_option_string");
+            _mpvGetPropertyString = (MpvGetPropertystring)GetDllType(typeof(MpvGetPropertystring), "mpv_get_property");
+            _mpvSetProperty = (MpvSetProperty)GetDllType(typeof(MpvSetProperty), "mpv_set_property");
+        }
+
+        public void Pause()
+        {
+            if (_mpvHandle == IntPtr.Zero)
+                return;
+
+            var bytes = Encoding.UTF8.GetBytes("yes\0");
+            _mpvSetProperty(_mpvHandle, Encoding.UTF8.GetBytes("pause\0"), MpvFormatString, ref bytes);
+        }
+
+        private void Play()
+        {
+            if (_mpvHandle == IntPtr.Zero)
+                return;
+
+            var bytes = Encoding.UTF8.GetBytes("no\0");
+            _mpvSetProperty(_mpvHandle, Encoding.UTF8.GetBytes("pause\0"), MpvFormatString, ref bytes);
+        }
+
+        public bool IsPaused()
+        {
+            if (_mpvHandle == IntPtr.Zero)
+                return true;
+
+            IntPtr lpBuffer = Marshal.AllocHGlobal(10);
+            _mpvGetPropertyString(_mpvHandle, Encoding.UTF8.GetBytes("pause\0"), MpvFormatString, ref lpBuffer);
+            return Marshal.PtrToStringAnsi(lpBuffer) == "yes";
+        }
+
+        public void SetTime(double value)
+        {
+            if (_mpvHandle == IntPtr.Zero)
+                return;
+
+            _mpvCommand(_mpvHandle, new[] { "seek", value.ToString(CultureInfo.InvariantCulture), "absolute", null });
+        }
+
+        private void buttonPlay_Click(object sender, EventArgs e)
+        {
+            if (_mpvHandle != IntPtr.Zero)
+                _mpvTerminateDestroy(_mpvHandle);
+
+            LoadMpvDynamic();
+            if (_libMpvDll == IntPtr.Zero)
+                return;
+
+            _mpvHandle = _mpvCreate.Invoke();
+            if (_mpvHandle == IntPtr.Zero)
+                return;
+
+            _mpvInitialize.Invoke(_mpvHandle);
+            _mpvSetOptionString(_mpvHandle, Encoding.UTF8.GetBytes("vo\0"), Encoding.UTF8.GetBytes("direct3d\0"));
+            _mpvSetOptionString(_mpvHandle, Encoding.UTF8.GetBytes("keep-open\0"), Encoding.UTF8.GetBytes("always\0"));
+            int mpvFormatInt64 = 4;
+            uint windowId = (uint)pictureBox1.Handle.ToInt64();
```

Comment:
Ups, surely a bug too (should just be an "int" I guess, so it uses the correct 32/64 bit int type accordingly to pointer size). Will fix it asap.